src/cegpy/graphs/_ceg.py

```diff
@@ -209,6 +207,8 @@ def _generate_dot_graph(self, edge_info="probability"):
         for (src, dst, label), attribute in edge_info_dict.items():
             if edge_info == "count":
                 edge_details = str(label) + "\n" + str(attribute)
+            elif edge_info == "none":
+                edge_details = str(label)
             else:
                 edge_details = f"{label}\n{float(attribute):.2f}"
```

Comment:
```suggestion
        for (src, dst, label), attribute in edge_info_dict.items():
            
            if not edge_info:
                edge_details = str(label)
            elif edge_info == "count":
                edge_details = str(label) + "\n" + str(attribute)
            else:
                edge_details = f"{label}\n{float(attribute):.2f}"
```

These are the changes we'd need to make if using `None`. The truth value of a `None` type is always `False` and for a non-None type will be `True`. So checking it as `if not edge_info` will only be `True` when `edge_info` is `None`. Let me know if doesn't make sense and I'll give a better example

Reply:
Another option is to do this:

```
        edge_details = str(label)        
        for (src, dst, label), attribute in edge_info_dict.items():

            if edge_info == "count":
                edge_details = edge_details + "\n" + str(attribute)
            else:
                edge_details = f"{edge_details}\n{float(attribute):.2f}"
```

Since `edge_details` always include `str(label)`, you can add initialise the `edge_details` variable with the label and then don't need to explicitly check for the `None` case.